templates/frontends-pipeline-pr-checks/cypress_e2e_tests.sh

```diff
@@ -17,12 +17,12 @@ docker run \
 docker cp -a . "${TEST_CONT}:/e2e/"
 
 docker exec -i $TEST_CONT sh -c "npm i"
-docker exec -i $TEST_CONT sh -c "npm run test:e2e"
+docker exec -i $TEST_CONT sh -c "npm run ci:cypress-e2e-tests"
```

Comment:
niptick: this will only work on nodes that have `docker` available but since there's no way to configure the agent externally to be anything different I believe it's not a concern

Reply:
This node also has a very outdated podman version so I don't want to unintentionally use that.

Reply:
ok, makes sense. thanks!